packages/DateTimePickerCommon/src/styles.ts

```diff
@@ -74,3 +74,17 @@ export const Selects = styled.div`
     margin-right: sm;
   }
 `
+
+export const FixAriaMessageStyle = css`
```

Comment:
maybe `fixAriaMessageStyle` is better because it's not a component?

Reply:
you're right 👍